src/Entity/EmbedButton.php

```diff
@@ -170,4 +171,49 @@ public function calculateDependencies() {
     return $this->dependencies;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function postSave(EntityStorageInterface $storage, $update = TRUE) {
+    parent::postSave($storage, $update);
+
+    $new_button_icon_uuid = $this->get('button_icon_uuid');
+    if (isset($this->original)) {
+      $unchanged = $this->original;
+      $old_button_icon_uuid = $unchanged->get('button_icon_uuid');
```

Comment:
Just make this `$old_button_icon_uuid = $this->original->get('button_icon_uuid')`. There's no need to define `$unchanged` then.